### PR TITLE
Update CHANGELOG.md

### PR DESCRIPTION
**Pull request by paddycarver**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,7 @@
 ## 1.0.1 (Unreleased)
+
+* Do the thing!
```
Comment (paddyforan) [inline CHANGELOG.md]: Maybe a different comment?

**Comment by catsby** [comment]
What sorcery is this?

**Comment by paddycarver** [comment]
What do you mean, @catsby?

**Comment by catsby** [comment]
I mean what is this PR? Is this a legit thing or are we testing a bot integration?

**Comment by paddycarver** [comment]
Everything in the `hashibot-test` is org is a HashiBot test and not a legit PR or issue. This one was testing what happens when different actions were taken on reviews--e.g., can we automate dismissing reviews if more code is pushed after the review was posted, so the reviewer can know they need to take another look at the PR.

**Comment by catsby** [comment]
Thanks for clarifying :smile:

**Pull request closed without merging**